### ISSUE TITLE
2.9: can not select different target folder for automatic picture uploads

### ISSUE THREAD

**Issue by thommierother** [owncloud/android#2294]
### Actual behaviour
Default setting uses /CameraUpload and /Video for automatic uploads from mobile devices. It is not possible to select a different sub-folder which already exists on the OC account and save this setting (e.g./Pictures/[Devicename])

### Expected behaviour
Selection of any folder as target for automatic uploads from Android device
 
### Steps to reproduce
1. click on "path for camera upload"
2. select sub-fodler from folder list
3. Burger menu has no option to save the selection
4. application menu still has default setting

Can this problem be reproduced with the official owncloud server?
(url: https://demo.owncloud.org, user: test, password: test)

not tested. I used our own server (10.0.9)

### Environment data
Android version: 8.0

Device model: 
Samsung Edge S7 (SM G935F)

Stock or customized system:
stock

ownCloud app version:
2.9 beta

ownCloud server version:
10.0.9 stable

#### Web server error log
```
Insert your webserver log here
```

#### ownCloud log (data/owncloud.log)
```
Insert your ownCloud log here
```

**Comment by jesmrec** [comment]
thanks for reporting, we will have a look!

**Comment by thommierother** [comment]
Hi Jesus,
I think we talked about that shortly, 2 wks ago in Nuremberg, I don't remember if that was you ;-) If you need additional info or have something to test, drop me a note

**Comment by davigonz** [comment]
Hi @thommierother , yes, we talked about this shortly in Nuremberg 😊. I don't get exactly what you mean, the behaviour you expect is **Selection of any folder as target for automatic uploads from Android device** and that is already possible, have a look at the next gif.

![oct-02-2018 16-41-13](https://user-images.githubusercontent.com/15261363/46356207-8d07e680-c662-11e8-881c-a2c767c5832a.gif)

Here, we are choosing one of our account folders as target for automatic uploads by pressing the **CHOOSE** button at the bottom of the view. 

Please give us more information if the steps of the previous gif do not work for you.

**Comment by davigonz** [comment]
Anyway, there's a bug in 2.9 beta which makes the app crash when selecting the target folder for camera uploads.

**Comment by thommierother** [comment]
ok, it was that crash I see on my device too. I will wait for the final

**Comment by davigonz** [comment]
Hi @thommierother , we have just released ownCloud `2.9 beta 2` for Android in PlayStore, can you please try it out and tell us if the crash is still present? Thanks

**Comment by davigonz** [comment]
> Saving the alternative path (instead of CameraUpload/) works now without crash, also the automatic upload. I will observe it the next days, but the first impression is positive ;-)

Glad to hear that, since this issue is only about the problem when selecting the camera uploads folder and is already fixed, I'm going to close it, if you detect any other weird behaviour in camera uploads, please open a new issue or have a look at the issues already opened with the label `sync::camera_uploads`. Thanks!
